index.js

```diff
@@ -1,92 +1,131 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+
+var counter = 1
```

Comment:
While `var` does let you change the data-type, I would say for consistency's sake, to try and use `let` or `const` for variable declaration!